### PR TITLE
Add images and which languages are available

### PR DESCRIPTION
**Pull request by PaulGirardin**
!!!!! STOP AND READ !!!!!

If the dropdown above says "base fork: lord/master", you are submitting your change to ALL USERS OF SLATE, not just your company. This is probably not what you want. Click "base fork" to change it to the right place.

If you're actually trying to submit a change to upstream Slate, please submit to our dev branch, PRs sent to the master branch are generally rejected.

### CODE REVIEW

File: source/includes/prestashop/configure.md [hunk 1]
```diff
@@ -6,14 +6,30 @@ To share your data with ShopRunBack, you must take the [authentication token of
```
Comment (leo-martin) [inline source/includes/prestashop/configure.md]: you must use

File: source/includes/prestashop/install.md [hunk 2]
```diff
@@ -10,6 +10,46 @@ First of all, you must create your own ShopRunBack account [here](https://dashbo
...
+
+Click on `Upload a module`.
+
+<img src="../../images/prestashop/ps1.7.2.5_modules-add-module.png" />
+
+Select the .zip you just downloaded.
+
+<img src="../../images/prestashop/ps_modules-select-zip.png" />
+
+The module is now installed. You can now click on `Configure` to direclty go to the module configuration page.
```
Comment (leo-martin) [inline source/includes/prestashop/install.md]: directly

File: source/includes/prestashop/use.md [hunk 3]
```diff
@@ -40,18 +40,34 @@ For instance, some of your data is automatically synchronized after some actions
...
 </aside>
 
-## How does my customers create a return demand?
+## Which language can I use?
 
-Any customer can create a return demand once an order is at least "Shipped".
+It is currently available in English (GB and US) and French.
 
-They then have a button to create a return demand on the details of the order.
+## How does my customers create a return request?
```
Comment (leo-martin) [inline source/includes/prestashop/use.md]: Do

File: source/includes/prestashop/configure.md (hunk 1 above)
Comment (tildedash) [reply source/includes/prestashop/configure.md]: remove the FR part of the URL, the user gonna be rejected to its preferred language

File: source/includes/prestashop/configure.md [hunk 4]
```diff
@@ -2,18 +2,34 @@
...
 
-To share your data with ShopRunBack, you must take the [authentication token of your ShopRunBack account](https://dashboard.shoprunback.com/fr/tokens).
+To share your data with ShopRunBack, you must use the [authentication token of your ShopRunBack account](https://dashboard.shoprunback.com/fr/tokens).
 
 Copy it and go back to your website.
 
-Click on the "ShopRunBack" tab in the left menu, go to "Configuration", paste your token in the "API Token" field and save your configuration.
+Click on the `ShopRunBack` tab in the left menu.
+
+<img src="../../images/prestashop/ps1.6.0.9_left-menu-srb.png" />
```
Comment (tildedash) [inline source/includes/prestashop/configure.md]: use the markdown syntax for images

File: source/includes/prestashop/configure.md [hunk 5]
```diff
@@ -23,4 +39,6 @@ After configuring your account, we recommend you to directly synchronize all you
 There are 2 modes:
 
 - Sandbox: It is a <b>test</b> environment. The data on this environment is reset every monday.
-- Production: It is where your customers' return demands are made. <b>This is real data</b>!
+- Production: It is where your customers' return requests are made. <b>This is real data</b>!
```
Comment (tildedash) [inline source/includes/prestashop/configure.md]: use markdown (**bold**)

**Pull request merged**